ironfish/src/wallet/account.ts

```diff
@@ -398,19 +416,19 @@ export class Account {
         await this.walletDb.saveDecryptedNote(this, spentNoteHash, spentNote, tx)
       }
 
-      await this.walletDb.saveTransaction(
-        this,
-        transaction.hash(),
-        {
-          transaction,
-          blockHash: null,
-          sequence: null,
-          submittedSequence,
-          timestamp: new Date(),
-          assetBalanceDeltas,
-        },
-        tx,
-      )
+      const transactionValue = {
+        transaction,
+        blockHash: null,
+        sequence: null,
+        submittedSequence,
+        timestamp: new Date(),
+        assetBalanceDeltas,
+      }
+
+      await this.saveConnectedMintsToAssetsStore(transactionValue, tx)
```

Comment:
nit - wondering if the `Connected` still makes sense in the name if we call this for pending transactions?